content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -375,6 +376,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `CSIMigrationGCEComplete`: Stops registering the GCE-PD in-tree plugin in kubelet and volume controllers and enables shims and translation logic to route volume operations from the GCE-PD in-tree plugin to PD CSI plugin. Requires CSIMigration and CSIMigrationGCE feature flags enabled and PD CSI plugin installed and configured on all nodes in the cluster.
 - `CSIMigrationOpenStack`: Enables shims and translation logic to route volume operations from the Cinder in-tree plugin to Cinder CSI plugin. Supports falling back to in-tree Cinder plugin if a node does not have Cinder CSI plugin installed and configured. Requires CSIMigration feature flag enabled.
 - `CSIMigrationOpenStackComplete`: Stops registering the Cinder in-tree plugin in kubelet and volume controllers and enables shims and translation logic to route volume operations from the Cinder in-tree plugin to Cinder CSI plugin. Requires CSIMigration and CSIMigrationOpenStack feature flags enabled and Cinder CSI plugin installed and configured on all nodes in the cluster.
+- `CSIVolumeSupportFSGroup`: Allows CSIDrivers to use the `supportsFsGroup` field. This field controls where volumes created by a CSIDriver support volume ownership and permission modifications when these volumes are mounted. 
```

Comment:
where -> whether?

Reply:
CSIVolumeSupportFSGroup sorts after CSIPersistentVolume; remember to adjust this to use the correct ordering.

Reply:
I've adjusted this to be `whether` instead of `where`, and corrected the ordering.